test/validators/transfer-bulk.ts

```diff
@@ -0,0 +1,137 @@
+    // Register validator with 7 operators
+    const { podId } = await helpers.registerValidators(4, 1, '90000', [1, 2, 3, 4, 5, 6, 7]);
+
+    // Transfer validator to an existing pod
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    ), [GasGroup.TRANSFER_VALIDATOR_EXISTED_POD]);
+  });
+
+  // SHOULD GIVE ERROR OF MAYBE INVALID TO POD INSTEAD OF ACCOUNT LIQUIDATABLE
+  it('Bulk transfer 10 validators to a non owned pod', async () => {
+    // Register validator with 7 operators
+    const { podId } = await helpers.registerValidators(5, 1, '90000', helpers.DataGenerator.pod.new(), [GasGroup.REGISTER_VALIDATOR_NEW_STATE]);
+
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('InvalidPod');
+  });
+
+  // SHOULD GIVE ERROR OF MAYBE INVALID FROM POD
+  it('Bulk transfer 10 validators to an invalid pod', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId.slice(0, -1) + 'a',
+      podResult1.podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('InvalidPod');
+  });
+
+  // SHOULD GIVE ERROR OF MAYBE VALIDATOR SHARE MISMATCH
+  it('Validator and share length mismatch', async () => {
+    // 10 validators and 11 shares
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      [podResult1.validators[0].publicKey, ...podResult2.validators.map((validator: any) => validator.publicKey)],
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length + 1).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('ValidatorShareMismatch');
+
+    // 9 validators and 10 shares
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[4]).bulkTransferValidators(
+      podResult2.validators.map((validator: any) => validator.publicKey),
+      podResult1.podId,
+      podResult3.podId,
+      Array(podResult2.validators.length).fill(helpers.DataGenerator.shares(0))
+    )).to.be.revertedWith('ValidatorShareMismatch');
+  });
+
+  // // NEED TO IMPLEMENT AN AMOUNT
+  // it('Transfer validator with not enough amount', async () => {
+
+  // });
+
+  // // NEED TO IMPLEMENT AN AMOUNT
+  // it('Transfer validator with not enough balance', async () => {
```

Comment:
Once amount is implemented then we can add not enough amount / not enough deposited tests